docs/user-guide/introduction.rst

```diff
@@ -48,3 +48,21 @@ you are interested. You can find all of our documentation at `our docs page <htt
 
 Many examples require the use of a type, and for those we will use "rpm". However,
 examples in this guide will only cover features that are common across content types.
+
+Platform Support Policy
+-----------------------
+
+CentOS/Red Hat Enterprise Linux (RHEL)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Pulp team provides RPM builds for the last two major releases of RHEL. For
```

Comment:
I don't think we reached consensus on this, and I still don't think we need this policy. Before el7, we only deployed on el6 because el5 was too old and difficult to support. We are also seriously considering dropping el6 support soon. Foreman has already announced deprecation of el6, which will of course apply to katello. Adding this policy would prevent us from following suit.

https://theforeman.org/manuals/1.12/index.html#Releasenotesfor1.12

"EL6 is now deprecated - while fully supported in 1.12, please migrate to EL7 for future releases."

Retaining the flexibility to drop support for an EL* at whatever time makes the most sense is valuable, and it only happens every few years.